inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,86 +1,412 @@
+
+NOTE: *Agreement:* The annotations of different annotators usually do not match perfectly. This aspect of difference / similarity is called agreement. For agreement, the measures provided in {product-name}, precision, recall and F1 are common.
+
+* *Agreement*: On this page, you can calculate the so called annotator agreement.
+*_Note:_* Only documents *marked as finished* by annotators (clicking on the
+little lock on the annotation page) are taken into account.
+
+* *Monitoring:* Here you can check the overall progress of your
+project; see which user is working on or has finished which document; and toggle for each user the status of each document between *Done / In Progress* or between *New / Locked*. For details, see <<Monitoring>> in the main documentation.
+
+* *Evaluation*: The evaluation page shows a learning courve diagram of
+each recommender (see xref:recommenders_in_getting_started[Recommender]).
+
+* *Settings*: Here, you can organize, manage and adjust all the details of your project. We had a look on those you need to get started for your own projects in the section <<Project Settings>> already.
+
+This was the overview on what you can _do_ in each project and what elements each project _has_. Now you are ready to go for your own annotations.
+
+
+== First Annotations with {product-name}
+
+
+In this section, we will make first annotations. If you have not downloaded and imported an example project yet, we recommend to return to <<Download and Import an Example Project>> and do so first.
+In this section, no or little theory and background will be explained. In case you want to have some theory and background knowledge first, we recommend reading the section <<Structure Of an Annotation Project>>.
+
+
+*Create your first annotations*
+
+You may want to watch our https://www.youtube.com/watch?v=wp4AN3p23mQ&list=PL5Hz5pttaj96SlXHGRZf8KzlYvpVHIoL-&index=3&t=0s[tutorial video „Overview“] on how to create
+annotations. But this guide, too, will lead you step by step. We will create a Named Entity annotation which tells whether a mention is a person (PER), location (LOC), organization (ORG) or other (OTH):
+
+NOTE: *Creating an Own Project:* In this guide, we will use our example project. If you would like to create an own project later on, click on *create*, enter a project name and click on *save*. Use the *Projects* link at the top of the screen to return to the project overview and select the project you just created to work with it. See <<Project Settings>> in order to add documents, users, guidelines and more to your project.
```

Comment:
Creating your own project

Reply:
an own project reads strangely

Reply:
I checked again (also o native speaker): There is no problem with it.